docs/spaces/index.asciidoc

```diff
@@ -2,28 +2,112 @@
 [[xpack-spaces]]
 == Spaces
 
-With spaces, you can organize your dashboards and other saved objects into meaningful categories.
-After creating your own spaces, you will be asked to choose a space when you enter Kibana. 
-Once inside a space, you will only see the dashboards and other saved objects that belong to that space. 
+Spaces enable you to organize your dashboards and other saved 
+objects into meaningful categories. Once inside a space, you see only 
+the dashboards and saved objects that belong to that space. 
 
-You can change your current space at any time, by clicking on the space avatar in the top left.
+{kib} creates a default space for you. 
+After you create your own 
+spaces, you're asked to choose a space when you enter Kibana. You can change your 
+current space at any time.
 
 [role="screenshot"]
 image::spaces/images/change-space.png["Change current space"]
 
-With security enabled, you can <<xpack-security-authorization, control which users have access to individual spaces>>.
+Kibana supports spaces in several ways.  You can:
 
+[[spaces-getting-started]]
+
+* <<spaces-managing, View&comma; create&comma; and delete spaces>>
+* <<spaces-control-feature-visibility, Control feature access based on user need>>
+* <<spaces-control-user-access, Control feature access based on user privilege>>
+* <<spaces-moving-objects, Move objects between spaces>>
+* <<spaces-delete-started, Disable the Spaces feature>>
 
 [float]
-[[spaces-getting-started]]
-=== Getting Started
+[[spaces-managing]]
+=== View, create, and delete spaces
+
+The **Management > Spaces** overview lists your spaces and provides actions 
+for you to create, edit, and delete spaces.
+
+[role="screenshot"]
+image::spaces/images/space-management.png["Space management"]
+
+[float]
+==== Create a space 
+
+You can create as many spaces as you like. Each space must have a unique 
+space identifier. This short text string becomes part of the 
+{kib} URL when you are inside that space. {kib} suggests a space identifier based 
+on the name of your space, but you can customize the identifier to your liking.
+You cannot change the space identifier once you create the space.
+
+{kib} also has an <<spaces-api, experimental API>> if you prefer to create spaces programatically.
+
+[float]
+==== Delete a spaces
+
+Deleting a space permanently removes the space and all of its contents. 
+You can't delete the default space, but you can customize it to your liking.
+
+[float]
+[[spaces-control-feature-visibility]]
+=== Control feature access based on user need
+
+You have control over which features are visible in each space. 
+For example, you might hide Dev Tools 
+in your "Executive" space, show Stack Monitoring only in your "Admin" space,
+and give certain users access to only Dashboard and Canvas. Controlling feature 
```

Comment:
> and give certain users access to only Dashboard and Canvas

This part reads to me like it belongs in the "Control feature access based on user privilege" section